data.md

```diff
@@ -0,0 +1,106 @@
+## How We Use Data
+
+_The outline and format of this page is adopted from [Projects by IF: How IF uses data](https://www.projectsbyif.com/how-if-uses-data/)._
+
+- [How We Use Data](#how-we-use-data)
+- [Our Websites and Hosted Services](#our-websites-and-hosted-services)
+    - [What data we collect](#what-data-we-collect)
+    - [Why we collect data](#why-we-collect-data)
+    - [Where we keep data](#where-we-keep-data)
+    - [How long we keep data](#how-long-we-keep-data)
+- [Our Social Media Accounts and Third-party Services](#our-social-media-accounts-and-third-party-services)
+
+---
```

Comment:
We don't do this line here https://raw.githubusercontent.com/hyphacoop/handbook/master/coronavirus.md

```suggestion
```